g2zproxy

```diff
@@ -179,12 +180,17 @@ class G2ZProxy(object):
                     # Get avarage value and time
                     data = [ v / len_data for v in reduce(lambda x, y: [ x[0] + y[0], x[1] + y[1] ], data) ]
                     # Update metric record
+		    logger.debug({'value': data[0], 'time': data[1] })
                     metric.update({ 'value': data[0], 'time': data[1] })
-
-        pool = Pool(processes=50)
-        pool.map(getData, self.metrics)
-        pool.close()
-        pool.join()
+	    try:
+		pool = Pool(processes=50)
```

Comment:
also please move this line before try:except. it should not cause exception, and we should keep try:except as small as possible